process/modules/rel_to_part.py

```diff
@@ -1,24 +1,28 @@
 from collections import defaultdict
 from typing import List
+
 import stanza
 import udon2
 import pyconll
 import pymorphy2
+
 from process.module import ParaphraseModule
 from process.preprocessing_utils import PreprocessingUtils
 
+
 class ReltoPart(ParaphraseModule):
     def __init__(self, name="rel_to_part") -> None:
         super().__init__(name=name)
     
     def load(self, preproc_utils: PreprocessingUtils) -> None:
-        # load any tools as `preproc_utils` attributes
-        self.loaded = True
+         if getattr(preproc_utils, 'morph', None) is None:
+             preproc_utils.morph = pymorphy2.MorphAnalyzer()
+         self.loaded = True
         
     def conjunction_check(self, tree_node, sent, morph):
-      head = morph.parse(str(tree_node.parent.parent).split('|')[-1])[0]
+      head = preproc_utils.morph.parse(str(tree_node.parent.parent).split('|')[-1])[0]
```

Comment:
```suggestion
      head = morph.parse(str(tree_node.parent.parent).split('|')[-1])[0]
```

и то же в L25, L34, L36